algorithms/gr-dij/python3/dijkstra.py

```diff
@@ -0,0 +1,83 @@
+# Library for INT_MAX
+import sys
+
+
+class Dijkstra:
+
+    def __init__(self, vertices):
+        self.V = vertices
+        self.graph = [[0 for column in range(vertices)]
+                      for row in range(vertices)]
+
+    def print_solution(self, dist):
+        print("Vertex Distance from Source")
+        for node in range(self.V):
+            print(node, "tt", dist[node])
+
+    # A utility function to find the vertex with
+    # minimum distance value, from the set of vertices
+    # not yet included in shortest path tree
+    def minDistance(self, dist, split_set):
+
+        # Initialize minimum distance for next node
+        min = sys.maxsize
+
+        # Search not nearest vertex not in the
+        # shortest path tree
+        for v in range(self.V):
+            if dist[v] < min and split_set[v] is False:
+                min = dist[v]
+                min_index = v
+
+        return min_index
+
+    # Function that implements Dijkstra's single source
+    # shortest path algorithm for a graph represented
+    # using adjacency matrix representation
+    def dijkstra(self, src):
+
+        dist = [sys.maxsize] * self.V
+        dist[src] = 0
+        split_set = [False] * self.V
+
+        for count in range(self.V):
```

Comment:
Codacy found an issue: [Unused variable 'count'](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6353492)

Reply:
it was needed because of print_solution function